eZ/Publish/Core/FieldType/User/Type.php

```diff
@@ -53,12 +53,13 @@ class Type extends FieldType
         ],
     ];
 
-    /**
-     * @param \eZ\Publish\Core\Persistence\Cache\UserHandler $userHandler
-     */
-    public function __construct(UserHandler $userHandler)
-    {
-        $this->userHandler = $userHandler;
+    /** @var \eZ\Publish\API\Repository\UserService */
```

Comment:
NOTE: Repo inside field type is effectively and indirect circular dependency.

Reply:
Besides that, really good to see movement on getting rid of having to use user service for content actions on user types ;)